dimmer.el

```diff
@@ -145,14 +144,16 @@ in ‘dimmer-face-color’."
       (setq dimmer-buffer-face-remaps nil))))
 
 (defun dimmer-filtered-buffer-list ()
-  "Get filtered subset of all buffers."
-  (seq-filter
-   (lambda (buf)
-     (let ((name (buffer-name buf)))
-       (not (or (eq ?\s (elt name 0)) ; leading space
-                (and dimmer-exclusion-regexp
-                     (string-match-p dimmer-exclusion-regexp name))))))
-   (buffer-list)))
+  "Get filtered subset of all visible buffers in the current frame."
+  (let (buffers)
+    (walk-windows
```

Comment:
This is new for me... Mind if I ask, is `walk-windows` more than syntactic sugar for mapping over `window-list`?

Reply:
It's slightly more than that. Take a look at the source, using `find-function` or `M-.` to jump to its definition if you have either a recent Emacs or my `elisp-slime-nav` package.

Reply:
ok, will do.